### PR TITLE
Bug fix in OVEP csharp sample

### PR DESCRIPTION
**Pull request by shamaksx**
Updated OVEP csharp sample for a bug fix

### CODE REVIEW

File: c_cxx/OpenVINO_EP/Linux/squeezenet_classification/squeezenet_cpp_app.cpp [hunk 1]
```diff
@@ -1,7 +1,6 @@
 /*
 Copyright (C) 2021, Intel Corporation
 SPDX-License-Identifier: Apache-2.0
-
```
Comment (sfatimar) [inline c_cxx/OpenVINO_EP/Linux/squeezenet_classification/squeezenet_cpp_app.cpp:4]: Please add this line

**Comment by sfatimar** [comment]
@jywu-msft Please review

File: c_cxx/OpenVINO_EP/Linux/squeezenet_classification/squeezenet_cpp_app.cpp (hunk 1 above)
Comment (shamaksx) [reply c_cxx/OpenVINO_EP/Linux/squeezenet_classification/squeezenet_cpp_app.cpp]: Line is added

**Comment by shamaksx** [comment]
> @shamaksx please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="intel"]

**Comment by shamaksx** [comment]
> @shamaksx the command you issued was incorrect. Please try again.
> 
> Examples are:
> 
> ```
> @microsoft-github-policy-service agree
> ```
> 
> and
> 
> ```
> @microsoft-github-policy-service agree company="your company"
> ```

@microsoft-github-policy-service agree company="intel"

**Review by jywu-msft — APPROVED** [review]

**Pull request merged**